### PR TITLE
Adds more wording fixes

### PR DESCRIPTION
**Pull request by Moejoe90**

### CODE REVIEW

File: app/components/kyc/personal/InstantId.tsx [hunk 1]
```diff
@@ -34,7 +34,7 @@ export const KycPersonalInstantIdComponent = injectIntlHelpers<
     steps={3}
     currentStep={3}
     title={formatIntlMessage("kyc.personal.instant-id.title")}
```
Comment (krzkaczor) [inline app/components/kyc/personal/InstantId.tsx]: cant you make title to take `ReactNode` as well and then use FormattedMessage and drop `formatIntlMessage` entirely?

File: app/components/settings/Settings.tsx [hunk 2]
```diff
@@ -15,44 +15,47 @@ interface IProps {
...
+    <LayoutAuthorized>
+      <Row className="row-gutter-top">
+        <Col xs={12}>
+          <SectionHeader>
+            <FormattedMessage id="settings.security-settings.title" />
+          </SectionHeader>
+        </Col>
         <Col lg={4} xs={12}>
-          <BackupSeedWidget />
+          <VerifyEmailWidget step={(i = i + 1)} />
```
Comment (krzkaczor) [inline app/components/settings/Settings.tsx]: cant you just use i++? (or ++i FWIW)

File: app/components/settings/Settings.tsx [hunk 3]
```diff
@@ -15,44 +15,47 @@ interface IProps {
...
-          <FormattedMessage id="settings.security-settings.title" />
-        </SectionHeader>
-      </Col>
-      <Col lg={4} xs={12}>
-        <VerifyEmailWidget />
-      </Col>
-
-      {isLightWallet && (
+export const SettingsComponent: React.SFC<IProps> = ({ isLightWallet }) => {
+  let i = 0;
```
Comment (krzkaczor) [inline app/components/settings/Settings.tsx]: please rename it to something more accurate like `securityStepCounter` or something?

File: app/components/settings/Settings.tsx [hunk 4]
```diff
@@ -15,44 +15,47 @@ interface IProps {
...
-);
+        )}
+
+        <Col lg={4} xs={12}>
+          <KycStatusWidget step={(i = i + 1)} />
+        </Col>
+
+        {process.env.NF_FEATURE_EMAIL_CHANGE_ENABLED === "1" && (
+          <>
+            {/* TODO: Remove message */}
```
Comment (krzkaczor) [inline app/components/settings/Settings.tsx]: what message?

File: app/components/settings/backupSeed/BackupSeedWidget.tsx [hunk 5]
```diff
@@ -11,6 +11,7 @@ import { Button } from "../../shared/Buttons";
 import { PanelDark } from "../../shared/PanelDark";
 import { settingsRoutes } from "../routes";
 
+import { compose } from "redux";
```
Comment (krzkaczor) [inline app/components/settings/backupSeed/BackupSeedWidget.tsx]: order imports according to guideliens please.

File: app/modules/settings/sagas.ts [hunk 6]
```diff
@@ -18,14 +20,33 @@ export function* addNewEmail(
...
   const user = yield select((s: IAppState) => selectUser(s.auth));
+  const walletType = yield select((s: IAppState) => selectWalletType(s.web3));
+
+  let addEmailMessage;
+
   //TODO: Add translations
   try {
+    switch (walletType) {
+      case WalletType.BROWSER:
+        addEmailMessage = "Please confirm on Metamask if the email address is correct";
```
Comment (krzkaczor) [inline app/modules/settings/sagas.ts]: this is missing translations

File: app/components/settings/Settings.tsx (hunk 2 above)
Comment (Moejoe90) [reply app/components/settings/Settings.tsx]: done

File: app/components/settings/Settings.tsx (hunk 3 above)
Comment (Moejoe90) [reply app/components/settings/Settings.tsx]: done

File: app/components/settings/Settings.tsx (hunk 4 above)
Comment (Moejoe90) [reply app/components/settings/Settings.tsx]: This is a comment from communication they want the title to be removed completely and messages

File: app/components/settings/backupSeed/BackupSeedWidget.tsx (hunk 5 above)
Comment (Moejoe90) [reply app/components/settings/backupSeed/BackupSeedWidget.tsx]: done

File: app/modules/settings/sagas.ts (hunk 6 above)
Comment (Moejoe90) [reply app/modules/settings/sagas.ts]: This is fine for now i want to do this in another PR that deals with all sagas and flows

**Pull request merged**